Sources/DolbyIOUIKit/Theme/Asset/ThemeManager.swift

```diff
@@ -7,12 +7,12 @@ import Foundation
 import SwiftUI
 
 public class ThemeManager: ObservableObject {
-    static public let shared: ThemeManager = ThemeManager()
+    static public let instance: ThemeManager = ThemeManager(theme: DefaultTheme())
```

Comment:
I think `shared` or `sharedInstance` are more commonly used naming in iOS. @aravind-raveendran Any comment?

Reply:
`shared` is the commonly used name.